cdk/index.ts

```diff
@@ -48,8 +48,15 @@ class StaticSiteStack extends cdk.Stack {
 }
 
 const app = new cdk.App();
-const environmentId = app.node.tryGetContext("EnvironmentId");
-const spaStack = new StaticSiteStack(app, environmentId + "StaticSpa", {
+const environmentParam = app.node.tryGetContext("EnvironmentId");
```

Comment:
Should we change this to:
```
const environmentParam = app.node.tryGetContext("EnvironmentId") ?? app.node.tryGetContext("TicketId");
```
To be consistent with atat-web-api?

Reply:
No. ``TicketId` was never a thing on this repo so there's no need to add that backwards compatibility. We'll let folks be broken for a little while on `atat-web-api` but there's no reason to add legacy code if that legacy never existed.